data/json/items/melee/swords_and_blades.json

```diff
@@ -24,14 +24,14 @@
     "symbol": "!",
     "color": "light_gray",
     "name": { "str": "nord" },
-    "description": "The nail sword, or nord for short.  This wooden sword has a dozen nails sticking at jagged angles from edge of the blade, making it much better at chopping than slashing.",
+    "description": "The nail sword, or nord for short.  This wooden sword has a dozen nails sticking at jagged angles from the edge of the blade, making it much better at chopping than slashing.",
     "material": [ "wood" ],
     "volume": "2600 ml",
     "weight": "1010 g",
     "longest_side": "100 cm",
     "bashing": 12,
     "cutting": 4,
-    "to_hit": { "grip": "solid", "length": "long", "surface": "every", "balance": "neutral" },
+    "to_hit": { "grip": "solid", "length": "long", "surface": "any", "balance": "neutral" },
     "price_postapoc": 100,
```

Comment:
I would leave this at "every", given the description of the nails. Since you're not relying on the wood itself to deal any damage, the nails should be the focus of the to_hit.

Reply:
Hey, thanks for looking.  The reason those are "any" instead of "every" is because "every" is seemingly supposed to be used for things like chainsaws or electrified weapons.
I'm not convinced a nail sword would do significant damage on a glancing blow, so I set it to "any" for now.

I referred to game_balance.md when adjusting the values.